example/interactive-cli.js

```diff
@@ -130,6 +144,54 @@ process.stdin.on('data', function(key) {
   } else if (key === '9') {
     client.setDebug(false);
     console.log('Debug messages are hidden');
+  } else if (key === 'hf') {
```

Comment:
Did you add all these shortcuts because you've found them useful? I'm not using this in the CLI, only via Node.js, so if it's useful to you, it's fine to add them.

Technically, this entire CLI section could be moved to its own project right? We could create this as a monorepo and add it there. What do you think?

Reply:
to move it out might be a good idea. And than build the cli tool from ground up again.